src/test/kotlin/org/rust/ide/inspections/RsLintLevelTest.kt

```diff
@@ -99,6 +99,20 @@ class RsLintLevelStructTest : RsInspectionsTestBase(RsStructNamingInspection::cl
         #[derive(Debug)]
         struct inhabitant;
     """)
+
+    fun testDirrectForbid() = checkByText("""
```

Comment:
Let's use `test dirrect forbid` style for test naming.
`testDirrectForbid` is old style that we don't use anymore.
If you want, you may convert all test names in this file to use new style

Reply:
Okay, I will convert all test names to the new style.

Reply:
And fix the `dirrect` typo please, it makes me nervous everytime I look at it! :D